postgresql/resource_postgresql_event_trigger.go

```diff
@@ -0,0 +1,408 @@
+				}
+			}
+
+			if values, ok := filter[eventTriggerFilterValueAttr]; ok {
+				var new_values []string
+
+				for _, value := range values.([]interface{}) {
+					new_values = append(new_values, pq.QuoteLiteral(value.(string)))
+				}
+
+				fmt.Fprint(b, " IN (", strings.Join(new_values, ","), ")")
+			}
+		}
+	}
+
+	eventTriggerFunction := d.Get(eventTriggerFunctionAttr).(string)
+	eventTriggerSchema := d.Get(eventTriggerFunctionSchemaAttr).(string)
+	fmt.Fprint(b, " EXECUTE FUNCTION ", pq.QuoteIdentifier(eventTriggerSchema), ".", eventTriggerFunction, "()")
+
+	createSql := b.String()
+
+	// Enable or disable the event trigger
+	b = bytes.NewBufferString("ALTER EVENT TRIGGER ")
+	fmt.Fprint(b, pq.QuoteIdentifier(eventTriggerName))
+
+	eventTriggerEnabled := d.Get(eventTriggerStatusAttr).(string)
+	fmt.Fprint(b, " ", eventTriggerEnabled)
+
+	statusSql := b.String()
+
+	// Table owner
+	b = bytes.NewBufferString("ALTER EVENT TRIGGER ")
+	eventTriggerOwner := d.Get(eventTriggerOwnerAttr).(string)
+	fmt.Fprint(b, pq.QuoteIdentifier(eventTriggerName), " OWNER TO ", eventTriggerOwner)
+
+	ownerSql := b.String()
+
+	// Start transaction
+	txn, err := startTransaction(db.client, d.Get(eventTriggerDatabaseAttr).(string))
+	if err != nil {
+		return err
+	}
+	defer deferredRollback(txn)
+
+	if _, err := txn.Exec(createSql); err != nil {
+		return err
+	}
+
+	if _, err := txn.Exec(statusSql); err != nil {
+		return err
+	}
+
+	if _, err := txn.Exec(ownerSql); err != nil {
+		return err
+	}
+
+	if err := txn.Commit(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func resourcePostgreSQLEventTriggerUpdate(db *DBConnection, d *schema.ResourceData) error {
+	eventTriggerName := d.Get(eventTriggerNameAttr).(string)
+	d.SetId(eventTriggerName)
+
+	// Enable or disable the event trigger
+	b := bytes.NewBufferString("ALTER EVENT TRIGGER ")
+	fmt.Fprint(b, pq.QuoteIdentifier(eventTriggerName))
+
+	eventTriggerEnabled := d.Get(eventTriggerStatusAttr).(string)
+	fmt.Fprint(b, " ", eventTriggerEnabled)
+
+	statusSql := b.String()
+
+	// Table owner
+	b = bytes.NewBufferString("ALTER EVENT TRIGGER ")
+	eventTriggerOwner := d.Get(eventTriggerOwnerAttr).(string)
+	fmt.Fprint(b, pq.QuoteIdentifier(eventTriggerName), " OWNER TO ", eventTriggerOwner)
+
+	ownerSql := b.String()
+
+	txn, err := startTransaction(db.client, d.Get(eventTriggerDatabaseAttr).(string))
+	if err != nil {
+		return err
+	}
+	defer deferredRollback(txn)
+
+	if _, err := txn.Exec(statusSql); err != nil {
+		return err
+	}
+
+	if _, err := txn.Exec(ownerSql); err != nil {
+		return err
+	}
+
+	if err := txn.Commit(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func resourcePostgreSQLEventTriggerDelete(db *DBConnection, d *schema.ResourceData) error {
+	eventTriggerName := d.Get(eventTriggerNameAttr).(string)
+	d.SetId(eventTriggerName)
+
+	b := bytes.NewBufferString("DROP EVENT TRIGGER ")
+	fmt.Fprint(b, pq.QuoteIdentifier(eventTriggerName))
+
+	sql := b.String()
```

Comment:
It's probably done in a similar way in other places but probably this could be simplified to

```suggestion
	sql := fmt.Sprintf("DROP EVENT TRIGGER %s", pq.QuoteIdentifier(eventTriggerName))
```

As you prefer